frontend/src/app/globals.css

```
@@ -1,28 +1,27 @@
+  --font-weight-body-small-strong: 700;
+  --line-height-body-small-strong: 1.4;
+
+  --border-radius-lg: 8px;
 }
 
-body {
-  background: var(--background);
-  color: var(--foreground);
-  font-family: Arial, Helvetica, sans-serif;
-} */
+@import "tailwindcss/utilities";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)